### ISSUE TITLE
Issue with volumePermissions

### ISSUE THREAD

**Issue by zerowebcorp** [jp-gouin/helm-openldap#147]
**Describe the bug**

When running on a bare metal k8s server and testing with a local volume mounted as hostPath, the bitnami non-root container is not able to access the PV even though the pv is created. The solution, as I understand is to use the `volumePermissions` init container. However, enabling this option throws an error. 

**To Reproduce**

```

volumePermissions:
  enabled: true
  image:
    registry: docker.io
    repository: bitnami/bitnami-shell
    tag: 10-debian-10
    pullPolicy: IfNotPresent

    command: ['sh', '-c', 'chmod -R g+rwX /bitnami']
  resources:
    requests: {}
```

**Expected behavior**

initContainer is created.

**Error**

```
➜  helm upgrade --install openldap helm-openldap/openldap-stack-ha -f values.yaml
Error: UPGRADE FAILED: failed to create resource: StatefulSet in version "v1" cannot be handled as a StatefulSet: json: cannot unmarshal object into Go struct field Container.spec.template.spec.initContainers.command of type []string
➜  
```

**Comment by zerowebcorp** [comment]
I did find this as a workaround

```
initContainers:
  - name: volume-permissions
    image: busybox
    command: [ 'sh', '-c', 'chmod -R g+rwX /bitnami' ]
    volumeMounts:
      - mountPath: /bitnami
        name: data
```

**Comment by jp-gouin** [comment]
Fix in `v4.2.2`

**Comment by phucly01** [comment]
@zerowebcorp Did setting the permission work for you?
I am getting chmod: Chaning permission of '/bitnami': Operation not permitted

Disabling pod security context didn't help
Adding containerSecurityContext under volume-permission didn't help.

Update: The volumePermission section isn't very useful.  I am not sure what is its intention but I need to change the volume permission under initContainer section.
